examples/gripping.py

```diff
@@ -0,0 +1,194 @@
+import os
+import pybullet as p
+import pybullet_data
+import pybullet_industrial as pi
+import numpy as np
+import time
+
+
+def build_circular_path(center, radius, min_angle, max_angle, steps, height):
```

Comment:
there is already a circular path builder that creates a toolpath object.
Plase use this instead.